test/functionality1/email-normalization.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+cd "$script_dir"
+
+source ../util/setup.sh
+
+set -xe
+
+q "delete from duo_session"
+q "delete from person"
+q "delete from onboardee"
+q "delete from undeleted_photo"
+
+q "insert into person (email, normalized_email, name, date_of_birth, coordinates, gender_id, about, unit_id) values ('user+1@example.com', 'user@gmail.com', 'user+1', '1997-05-30', 'POINT(0 0)', 2, 'cool', 1)"
+q "insert into person (email, normalized_email, name, date_of_birth, coordinates, gender_id, about, unit_id) values ('user+2@example.com', 'user@gmail.com', 'user+2', '1997-05-30', 'POINT(0 0)', 2, 'cool', 1)"
+
+q "insert into person (email, normalized_email, name, date_of_birth, coordinates, gender_id, about, unit_id) values ('otheruser+1@example.com', 'otheruser@gmail.com', 'otheruser+1', '1997-05-30', 'POINT(0 0)', 2, 'cool', 1)"
```

Comment:
There's a `create-user.sh` script that can be used in place of these lines. Please use that and then update the email addresses of the new users

Reply:
Could you create them as `user1` and `user2`, for example, then change them to `user+1` and and `user+2`? Sorry to be pedantic about this, but I want to minimize the use of direct queries in tests as much as possible. I want the tests to test behavior as much as possible, rather than implementation details. Updating one field instead of inserting entire rows feels safer to me

Reply:
Makes sense, is this good?